source/features/linkify-action.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'dom-chef';
+
+import * as pageDetect from 'github-url-detection';
+
+import {ArrowLeftIcon} from '@primer/octicons-react';
+
+import features from '../feature-manager.js';
+import api from '../github-helpers/api.js';
+import observe from '../helpers/selector-observer.js';
+
+async function getPullRequestNumber(user: string, repository: string, runDatabaseId: string): Promise<number | false> {
+	const {resource} = await api.v4(`
+		query getPullRequestNumber($resource: URI!) {
+			resource(url: $resource) {
+				... on WorkflowRun {
+					event
+					checkSuite {
+						matchingPullRequests(last: 1) {
+							edges {
+								node {
+									number
+								}
+							}
+						}
+					}
+				}
+			}
+		}
+	`, {
+		variables: {
+			resource: `/${user}/${repository}/actions/runs/${runDatabaseId}`,
+		},
+	});
+
+	if (resource.event !== 'pull_request') {
+		return false;
+	}
+
+	const {edges} = resource.checkSuite.matchingPullRequests;
+	if (edges.length === 0) {
+		return false;
+	}
+
+	return edges[0].node.number;
+}
+
+async function add(contextBar: HTMLDivElement): Promise<void> {
+	const url = new URL(location.href);
+	if (typeof url.searchParams.get('pr') === 'string') {
```

Comment:
This is a static check that can be part of the `exclude` array in `features.add()`

```js
exclude: [
	 new URLSearchParams(location.search).has('param');
]
```

However… why are these being excluded? Can you provide an example?

Reply:
I'm on mobile.
Please check this comment! https://github.com/refined-github/refined-github/issues/6206#issuecomment-1566090775

Reply:
Ohh, that explains the choices made here.